insights/combiners/os_release.py

```diff
@@ -0,0 +1,136 @@
+"""
+OSRelease
+=========
+The ``OSRelease`` combiner uses the following parsers to try to identify if the
+current host is installed with a "Red Hat Enterprise Linux" system.
+
+    - :py:class:`insights.parsers.os_release.OsRelease`
+    - :py:class:`insights.parsers.redhat_release.RedhatRelease`
+    - :py:class:`insights.parsers.installed_rpms.InstalledRpms`
+
+It provides an attribute `is_rhel` that indicates if the host is "RHEL" or not.
+It also provides an attribute `product` which returns the estimated OS name of
+the system, "Unknown" will be returned by default when cannot identify the OS.
+"""
+from insights.core.plugins import combiner
+from insights.parsers.installed_rpms import InstalledRpm, InstalledRpms
+from insights.parsers.os_release import OsRelease
+from insights.parsers.redhat_release import RedhatRelease
+from insights.parsers.uname import Uname
+
+RHEL_STR = "Red Hat Enterprise Linux"
+RHEL_KEYS = ['rhel', 'red hat enterprise linux']
+"""Keywords in 'os-release' or 'redhat-release' that indicates an RHEL system."""
+NON_RHEL_KEYS = ['oracle', 'suse', 'sles', 'fedora', 'centos']
+"""Keywords in 'os-release' or 'redhat-release' that indicates a NON-RHEL system."""
+MIN_RHEL_PKGS = [
+    'audit-libs',
+    'basesystem',
+    'bash',
+    'coreutils',
+    'dbus',
+    'dmidecode',
+    'dracut',
+    'filesystem',
+    'firewalld',
+    'glibc',
+    'gmp',
+    'krb5-libs',
+    'libacl',
+    'libgcc',
+    'libselinux',
+    'NetworkManager',
+    'openssl-libs',
+    'passwd',
+    'systemd',
+    'util-linux',
+    ('dnf', 'yum'),
+    ('redhat-release', 'redhat-release-server'),
+]
+"""Must-install packages for minimum installed RHEL system."""
```

Comment:
I see some comments are under the code line and some are above. Move all comments above the corresponding code line.

Reply:
Yeap, comment (started with `#` is always above the code) but docstring (wrapped with `""""""`) should be under the line, this is following the Sphinx style https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google